packages/flex-plugin-scripts/src/clients/baseClient.ts

```diff
@@ -22,21 +26,35 @@ export default abstract class BaseClient {
     return `https://${subDomain}${realmDomain}.twilio.com/${version}`;
   }
 
+/**
+ * Constructs user agent with core
+ * plugin builder packages
+ */
+  public static getUserAgent(packages: string[] = FLEX_PACKAGES): string {
+    return getPackageDetails(packages)
+            .reduce((userAgentString, pkg) =>
+                `${userAgentString} ${pkg.name}/${pkg.found ? pkg.package.version : '?'}`, '')
```

Comment:
use the new optional chaining: `${userAgentString} ${pkg.name}/${pkg.package?.version || '?'}`

Reply:
So the entire user agent is just package/version? Should we not prefix it all with something else, like `Flex Plugin Builder :: package/version package/version package/version`?

Reply:
Also, prefer `package@version` instead of `package/version` to be more consistent with npm notation.

Reply:
optional chaining: prefer not to, ternary and inspection of `found` before indexing into optional field uses the api (found boolean), is clearer, more defensive.

prefix + package@version notation: Not sure either will make a difference but following http spec for user agent format https://tools.ietf.org/html/rfc7231#section-5.5.3